Assets/FrostedGlass/Scripts/CommandBufferBlur.cs

```diff
@@ -2,9 +2,12 @@
 using UnityEngine.Rendering;
 
 [ExecuteInEditMode]
+[ImageEffectAllowedInSceneView]
 [RequireComponent(typeof(Camera))]
 public class CommandBufferBlur : MonoBehaviour
 {
+    private static Shader _STATIC_SHADER;
```

Comment:
Can you explain the use case for this ?

Reply:
When the [ImageEffectAllowedInSceneView] annotation is added to the behaviour, a new instance of the behaviour gets created behind-the-scenes on the SceneView Camera. However, for some reason in testing, I found that this new instance wasn't taking the serialized Shader property from the Main Camera, even when the appropriate Shader was set as the default parameter on the script file in the Project, so the _Shader member was always null. Adding a static reference that gets populated on Validate of the Main Camera's instance, solves this issue.

Reply:
Since the CommandBufferBlur Behaviour has a dependency on the **SeparableBlur.shader** asset, maybe a Shader.Find(""Hidden/SeparableGlassBlur") + Exception thrown in case the shader is not found would be better ? Thus making the _Shader property private.

Reply:
Makes sense. Happy to make that change.

Reply:
Please feel free to make it 👍

Reply:
Amended